Cargo.toml

```diff
@@ -28,7 +28,7 @@ heck = "0.4.0"
 zip = { version = "0.5.13", default-features = false }
 parity-wasm = "0.42.2"
 cargo_metadata = "0.14.1"
-codec = { package = "parity-scale-codec", version = "2.3.1", features = ["derive"] }
+scale = { package = "parity-scale-codec", version = "2.3.1", features = ["derive"] }
```

Comment:
Renamed because I have an embedded contract for testing purposes and `ink!` generates code with `::scale` imports.